doc.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
```

Comment:
Maybe we could have another command to make it easier for local development - which runs all three of the below command in sequence? e.g.  `./doc.sh serve_local`

Reply:
yep that would work, right now the commands just check if the proceeding step has already been run and print something if it's not been, it could be made to just run the missing steps if needed. So a ./doc.sh run would first create the docker image, then build the app, then run the app

Reply:
I've been thinking more about this and I dont think it's necessary now. `./doc.sh image` only needs to be run once. Then `./doc.sh build` and `./doc.sh run` correspond directly to `npm run build` and `npm run dev` You could even create package.json scripts to invoke ./doc.sh for those commands if you wanted. But then a local install of npm becomes a requirement for the project